.discourse-compatibility

```diff
@@ -0,0 +1 @@
+2.9.0.beta3: f8f5bacdf58a516c1a9000246587a7c09685f37e
```

Comment:
tests-passed is currently on `2.9.0.beta3` so this will lock tests-passed sites to this commit. Is this what you want to do?

Reply:
Yup :) I'm aware of that but since we do not have a better workaround, we'll just have to wait until stable is bumped to 2.9.0.beta4